### PR TITLE
Vulkan utilities version bump

### PR DESCRIPTION
**Pull request by MP2E**
###### Motivation for this change
Updates vulkan-loader, vulkan-headers, and vulkan-tools to their latest applicable versions

###### Things done
- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/libraries/vulkan-loader/default.nix [hunk 1]
```diff
@@ -13,8 +13,8 @@ stdenv.mkDerivation rec {
   src = fetchFromGitHub {
     owner = "KhronosGroup";
     repo = "Vulkan-Loader";
-    rev = "b02f64293680c484e1d7ff6ecb88f89277c0dc8c";
-    sha256 = "1n4vjyxlmi2ygx34srwbvalc5gz95gcsrmdw0k10353xja755gmj";
+    rev = "15fa85d92454f7823febeb68b56038d427e2a7a4";
```
Comment (pSub) [inline pkgs/development/libraries/vulkan-loader/default.nix]: There is a git tag for that release.

File: pkgs/development/libraries/vulkan-headers/default.nix [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+  version = "1.1.101";
 
   buildInputs = [ cmake ];
 
   src = fetchFromGitHub {
     owner = "KhronosGroup";
     repo = "Vulkan-Headers";
-    rev = "2fd5a24ec4a6df303b2155b3f85b6b8c1d56f6c0";
-    sha256 = "0cj4bd396qddh3nxvr7grnpfz89g3sbvm21cx4k3ga52sp1rslpb";
+    rev = "8e2c4cd554b644592a6d904f2c8000ebbd4aa77f";
```
Comment (pSub) [inline pkgs/development/libraries/vulkan-headers/default.nix]: There is a git tag for that release.

**Review by pSub — CHANGES_REQUESTED** [review]

**Comment by dywedir** [comment]
Slightly earlier PR: https://github.com/NixOS/nixpkgs/pull/55959

**Comment by Infinisil** [comment]
Superseded by above PR

**Pull request closed without merging**